cmd/activator/main.go

```diff
@@ -68,9 +68,6 @@ import (
 const (
 	component = "activator"
 
-	// Add enough buffer to not block request serving on stats collection
-	requestCountingQueueLength = 100
```

Comment:
\o/

Reply:
Only downside of this: there's now no buffering so when the mux is locked (during report) we'll block requests. I _think_ that's ok, bc report should be very fast, and only every second, but I wanted to say that out loud and double-check we're ok with that change.

Reply:
I personally am aware and okay with that. We've pushed the expensive ops (channel write) outside of the critical path there already so I think we should be good.

FWIW, this is the same as we already have in the QP, with the slight difference that this is across revisions and not just one revision.

Reply:
yeah, sounds legit to me, report doesn't really do anything expensive now so 🤷.